### PR TITLE
Portfolio: Add a contact form and finish the mobile version

### PR DESCRIPTION
**Pull request by HenaryKhuraijam**
# Portfolio: Add a contact form 
- Created new Branch
- Added contact form

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -68,6 +78,227 @@ body {
...
+  background-repeat: no-repeat;
+}
+
+footer .content-placeholder {
+  text-align: center;
+  padding: 60px 5px;
+}
+
+#contact-msg,
+#contact-me { color: #fff; }
```
Comment (ABDULALI3468) [inline style.css]: Please consider making the styles in accordance with the template to make it look alike the template.

## Solution:---

```
    font-size: 40px;
    line-height: 52px;
```

File: style.css [hunk 2]
```diff
@@ -68,6 +78,227 @@ body {
...
+  background-position: center top, right top;
+  background-repeat: no-repeat;
+}
+
+footer .content-placeholder {
+  text-align: center;
+  padding: 60px 5px;
+}
+
+#contact-msg,
```
Comment (ABDULALI3468) [inline style.css]: Please consider making the styles in accordance with the template to make it look alike the template.

## Solution:---

```
    font-size: 20px;
    line-height: 28px;
    width: 327px;
```

File: style.css [hunk 3]
```diff
@@ -68,6 +78,227 @@ body {
...
+#contact-me { color: #fff; }
+
+form {
+  width: 100%;
+  background-image: url(images/contact-bg_02.png);
+  background-position: right bottom;
+  background-repeat: no-repeat;
+  flex-wrap: wrap;
+}
```
Comment (ABDULALI3468) [inline style.css]: Please consider making the styles in accordance with the template to make it look alike the template.

## Solution:---

```
font-family: 'Poppins';
```

File: index.html [hunk 4]
```diff
@@ -1,39 +1,204 @@
...
+      <p id="contact-msg">
+        If you have an application you are interested in developing, a feature
+        that you need built or a project that needs coding. I’d love to help
+        with it
+      </p>    
+      <form class="form flex" action="https://formspree.io/f/mqkoalvp"
+      method="POST"
+      enctype="multipart/form-data">
+        <input class="form-input input-line" type="text" placeholder="Enter your name" name="name" maxlength="30" required/>
+        <input class="form-input input-line" type="email" id="email" name="email" placeholder="yourid@gmail.com" required/>
```
Comment (ABDULALI3468) [inline index.html:197]: Please consider making the styles in accordance with the template to make it look alike the template.

## Solution:---

```
    font-size: 17px;
    line-height: 20px;
```

File: index.html [hunk 5]
```diff
@@ -1,39 +1,204 @@
...
+        <textarea class="form-input" rows="10" maxlength="500" placeholder="Write your message here" name="message" required ></textarea>
```
Comment (ABDULALI3468) [inline index.html]: Please consider making the styles in accordance with the template to make it look alike the template.

## Solution:---

```
    font-size: 15px;
    line-height: 24px;
```

**Review by ABDULALI3468 — CHANGES_REQUESTED** [review]
# Changes Required 👍 

Hi @HenaryKhuraijam , 



Indeed you have done an exceptional job but still, there are some issues that need to be addressed to complete this project.


## Highlights 
✅All linters are checked.
✅Project is professionally documented.
✅The right GitHub flow method has been used.
✅Professional pull request title and description.


 ## Required Changes ♻️

> **_Check the comments below to have a look at the required changes._**

After implementing all the changes check again if your styles are in accordance with the design and matching! and then request another review with the same Pull Request.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

## Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification.

> * This is My **GitHub**: GitHub followers
> * This is My **Linkedin**: [Linkedin Badge](https://www.linkedin.com/in/abdul-ali-5400bb216/)&nbsp;
> * This is My **Twitter**: [Twitter Badge](https://twitter.com/mrabdul_ali)&nbsp;

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by HenaryKhuraijam** [comment]
# Make changes as suggested.
- Added font-size and line-height

File: index.html (hunk 5 above)
Comment (HenaryKhuraijam) [reply index.html]: Thank you for finding out my mistakes.

File: style.css [hunk 6]
```diff
@@ -68,6 +78,270 @@ body {
...
+footer .content-placeholder {
+  text-align: center;
+  padding: 60px 5px;
+}
+
+#contact-msg {
+  color: #fff;
+  font-size: 20px;
+  line-height: 28px;
+  width: 327px;
```
Comment (Zuhaib042) [inline style.css:287]: - [OPTIONAL] Kindly consider removing this line as we want this text to be responsive, not stationary having `327px` fixed width. so consider removing it and instead placing `text-align: center;` here to have this element centered but responsive.

**Review by Zuhaib042 — APPROVED** [review]
Hi @HenaryKhuraijam , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
approve

## Highlights
✅All linters are checked.
✅Project is professionally documented.
✅The right GitHub flow method has been used.
✅Professional pull request title and description.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**